Remove self.bot.prefix

gaggle opened the pull request:
`self.bot` doesn't exist

This is a simple proposal for #1368. Maybe there's a better way to do it?

gaggle commented:
Not immediately recognizing the failure, if anyone wants to guide me a bit I'm happy to iterate on the solution

sijis commented:
I think the prefix should be included in the output.

I don't know the path offhand but finding an example usage of self.bot.prefix should indicate how you can get acces to the attributes.

sijis commented:
I was finally able to poke around a little.
It seems that the way to get the same behaviour is doing: `self.plugins['Help']._bot.prefix`. This is not ideal but it accomplishes the same intended behaviour.

sijis commented on errbot/plugin_manager.py:
```diff
@@ -356,7 +356,7 @@ def activate_non_started_plugins(self):
                 if self.is_plugin_blacklisted(name):
                     errors += f'Notice: {plugin.name} is blacklisted, ' \
-                              f'use {self.bot.prefix}plugin unblacklist {name} to unblacklist it.\n'
+                              f'use plugin unblacklist {name} to unblacklist it.\n'
```
Based on earlier findings, this should work.
```suggestion
                              f'use "{self.plugins["Help"]._bot.prefix}plugin unblacklist {name}" to unblacklist it.\n'
```

gaggle commented:
Sorry @sijis I forgot this PR, thanks for your suggestion! I've implemented it, and rebased to latest upstream master. I get test-errors in `commands_test` but I'm not seeing their relation to my change. But open for feedback..

craigds commented:
Will there be a release for this fix soon?

sijis commented:
@craigds @gaggle The latest 6.1.7 release should include this fix.
